src/stories/Library/event-list-item/event-list-item.scss

```
@@ -100,27 +141,52 @@ $_desktop-content-rows: 5;
   flex-direction: column;
   align-items: flex-end;
   justify-content: flex-end;
-  row-gap: 8px;
   @include typography($typo__body-placeholder);
 }
 
+.event-list-item-stacked {
+  @include event-layout;
+  @include event-list-item-stacked-shadow-box;
+  padding-top: $s-sm;
+  padding-bottom: $s-sm;
+  grid-template-areas: "contentStacked";
+  align-items: center;
+}
+
+.event-list-item-stacked__content {
+  @include typography($typo__body-placeholder);
```

```suggestion
  @include typography($typo__small-caption);
```

The way i see it, this font should be `$typo__small-caption`. 

The font sizes on the current figma for this, is not correct. The size on the stacked__content should be the same as the description on the eventlist item. This is using __small-caption. The reason the design for this file is using 14px for the description and not 12px, is because of a temporary change that is not accepted yet. 

`$typo__body-placeholder is not correct, since this is even higher (16px), i would advice we use the same as was used in the original description for event-list-item, `$typo__small-caption`

if this is used elsewhere, remember to change that there too.